lib/util.js

```diff
@@ -446,7 +447,11 @@ function formatProperty(ctx, value, recurseTimes, visibleKeys, key, array) {
     }
   }
   if (!hasOwnProperty(visibleKeys, key)) {
-    name = '[' + key + ']';
+    if (isSymbol(key)) {
+      name = '[' + ctx.stylize(key.toString(), 'symbol') + ']';
+    } else {
+      name = '[' + key.toString() + ']';
```

Comment:
Well, a key could be either a symbol or a string. Anyway, removed `toString`, it's not needed and wasn't there in the first place. Not sure why symbols `toString` is not called automatically.